src/backend/main/go.main/facilities/REST/contacts.go

```diff
@@ -279,6 +281,41 @@ func (rest *RESTfacility) ContactExists(userID, contactID string) bool {
 	return rest.store.ContactExists(userID, contactID)
 }
 
+// Process a vcard file and create related contacts
+func (rest *RESTfacility) ImportVcardFile(info *UserInfo, file io.Reader) error {
+	vcards, err := contact.ParseVcardFile(file)
```

Comment:
if `contact.ParseVcardFile` returns an empty array it's not reported as an error, it should ?

Reply:
done in #1a14ef7c3